build/configs/rtl8730e/tc/defconfig

```diff
@@ -1317,11 +1317,6 @@ CONFIG_EXAMPLES_SECURITY_API_TEST=y
 # CONFIG_EXAMPLES_SMARTFS_POWERCUT is not set
 # CONFIG_EXAMPLES_ST_THINGS is not set
 CONFIG_EXAMPLES_TESTCASE=y
```

Comment:
Regardless of this PR,
this defconfig is `tc`, but no sub-tc is enabled.

Reply:
I understand you point :P
What do you suggest please?

Reply:
Let's enable kernel tc.

Reply:
Done